packages/@o3r/testing/package.json

```diff
@@ -6,85 +6,87 @@
     "testing",
     "otter"
   ],
+  "main": "dist/public_api.js",
```

Comment:
`dist` should not be needed as in the published package we dont have a dist folder.

Reply:
It is updated during the postbuild task which goes over all the dist reference and remove them

Reply:
Well it does not change anything after tests, I will leave it this way :)